static/panes/compiler.ts

```diff
@@ -2353,6 +2386,7 @@ export class Compiler extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Co
         this.filters = new Toggles(this.domRoot.find('.filters'), patchOldFilters(state.filters));
 
         this.optButton = this.domRoot.find('.btn.view-optimization');
+        this.stackUsageButton = this.domRoot.find('.btn.view-stack-usage');
```

Comment:
Let's hide the button when the compiler doesn't support it. This is done at line 2685 for the opt button with `this.optButton.toggle(!!this.compiler.supportsOptOutput);`.